auslib/db.py

```diff
@@ -1745,6 +1745,23 @@ def hasPermission(self, username, thing, action, product=None, transaction=None)
         return True
 
 
+class Dockerflow(AUSTable):
+    def getWatchdogValue(self, transaction=None):
+        return self.select(transaction=transaction)
+
+    def incrementWatchdogValue(self, changed_by, transaction=None, dryrun=False):
+        if not self.db.hasPermission(changed_by, "dockerflow", "modify", transaction=transaction):
+            raise PermissionDeniedError("%s is not allowed to modify dockerflow" % changed_by)
```

Comment:
Per IRC discussions, let's get rid of this. Because there's no reading of args from the HTTP request, there shouldn't be any opportunity for an injection attack.